RecoBTag/MXNet/python/Parameters/ParticleNet/V00/pfMassDecorrelatedParticleNetPreprocessParams_cfi.py

```diff
@@ -0,0 +1,246 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Maybe we can keep it like this for now and move them to the data files in a follow-up PR?

Reply:
given already submitted backports of this PR and unclear motivation for backports of future updates, this may be not that temporary. So, better not postpone.